### DISCUSSION TITLE
v0.90.5-beta

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#6057] [Releases]
## What's Changed

This release brings a few quality-of-life improvements, as well as bugfixes. The main highlight, however, is the increased support for localization as well as a new note type.

🌍 Internationalization
-----------------------

Preliminary support for internationalization has been added, thanks to the efforts of `@Nriver`. More specifically, the following languages now have a partial translation:

*   Spanish by `@hasecilu`
*   Chinese by `@Nriver`
*   Romanian.

Note that only the client has been translated so far and there are still a lot of tasks to handle before the application is fully translated.

💡 New note type: Mind map
--------------------------

Mind maps are usually a fast method to write down ideas and do brainstorming with a team. TriliumNext introduces this new note type with the help of [Mind Elixir](https://mind-elixir.com/), an open-source library.

Note that the integration is still in its infancy, so there might be bugs and a lack of feature until it is stabilized. The library itself supports quite a few features that are not implemented yet in Trilium: math expressions using KaTeX, images, links. Those might be implemented along the way, provided they are needed by users.

![v0 90 5-beta_image](https://github.com/user-attachments/assets/25f916f8-a91c-4007-9f19-1ded66818870)

> [!NOTE]
> For the note type to fully work (such as shared notes), it is ideal to update the server instance as well, if you are using one.

⚙️ Builds
---------

The internal build tooling has been updated, resulting in the following changes:

*   For macOS users there is now a `.dmg` installation method as well. `@JYC333`
    *   There is also an ARM-native version for macOS that can be used, however it will require [additional steps to get it running](https://github.com/TriliumNext/Notes/issues/329) due to the lack of notarization (to bypass the “TriliumNext Notes.app” is damaged error).
*   For both Windows and Linux we now have `amd64` builds as well.
    *   Please note these versions are not being actively tested due to lack of equipment, so feel free to raise any issues you might be encountering.

On the Docker side, [brought back Alpine-based Docker containers for amd64](https://github.com/TriliumNext/Notes/pull/366) by `@perfectrain`.

### 🐞 Bugfixes

*   [v0.90.4 docker does not read USER\_UID and USER\_GID from environment](https://github.com/TriliumNext/Notes/issues/331)
*   [Invalid CSRF token on Android phone](https://github.com/TriliumNext/Notes/issues/318)
*   [Excess spacing in lists](https://github.com/TriliumNext/Notes/issues/341)
*   [scrollbar-color makes the scrollbar appear natively styled](https://github.com/TriliumNext/Notes/issues/350)
*   Invisible scrollbar on Firefox
*   [Issue with note title text box border](https://github.com/TriliumNext/Notes/issues/358)
*   [Focus not set to input field when clicking `Include Note` from Block Editor](https://github.com/TriliumNext/Notes/issues/365)
*   [Fix a bug of find widget](https://github.com/TriliumNext/Notes/pull/377) by `@SiriusXT`
*   ['A JavaScript error occurred in the main process' when launching Trilium](https://github.com/TriliumNext/Notes/issues/368) (improved error handling).
    
*   [Note Tooltip isn't removed when clicking on internal trilium link in read-only mode](https://github.com/TriliumNext/Notes/issues/375)
    
*   [Calendar dropdown won't close if click/right-click other button that open notes from launcher bar](https://github.com/TriliumNext/Notes/issues/384)
    

### ✨ Improvements

*   [Improved the calendar button in the launch bar](https://github.com/TriliumNext/Notes/issues/374), adding support for easier month and year selection.
*   [Make first day of week configurable](https://github.com/TriliumNext/Notes/issues/247) (supports Sunday and Monday)
    *   Can be adjusted in Options → Appearance → Localization → First day of the week.
    *   The option is synced with the server and the client updates immediately.
*   Removed hard-coded libraries such as FancyTree and Bootstrap. This will later allow us to upgrade to the latest versions. `@JYC333`
*   [Hide Electron-specific settings in the web version](https://github.com/TriliumNext/Notes/issues/345)
*   [Add a toggle to promote the current note to a template](https://github.com/TriliumNext/Notes/issues/348)
*   Disable share toggle when in options
*   [Open New window from taskbar](https://github.com/TriliumNext/Notes/pull/373) by `@SiriusXT`
*   Pressing F2 to edit the branch prefix will now work only in the note tree to avoid issues with other interactive elements such as the mind map, but also since the option itself is not the most used ones to require a global shortcut.

New Contributors
----------------

*   [@JYC333](https://github.com/JYC333) made their first contribution in [#294](https://github.com/TriliumNext/Notes/pull/294)
*   [@hasecilu](https://github.com/hasecilu) made their first contribution in [#349](https://github.com/TriliumNext/Notes/pull/349)
*   [@SiriusXT](https://github.com/SiriusXT) made their first contribution in [#377](https://github.com/TriliumNext/Notes/pull/377)

**Full Changelog**: https://github.com/TriliumNext/Notes/compare/v0.90.4...v0.90.5-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/TriliumNext/Notes/releases/tag/v0.90.5-beta'>v0.90.5-beta</a>.</em>

**Comment by hasecilu** [comment]
Hi, I tried to update to both `latest` and `v0.90.5-beta` tags but it seems there is no build for `linux/amd64` architechture in both GitHub and Docker. Only are 2 ARM builds.
```shell
❯ docker pull ghcr.io/triliumnext/notes:latest
latest: Pulling from triliumnext/notes
no matching manifest for linux/amd64 in the manifest list entries
```

**Reply by Kofl** [reply]
Can confirm the issue also for new installs

**Reply by Kofl** [reply]
v0.90.6-beta solved the issue.

**Comment by SiriusXT** [comment]
Hi, could you please add the TriliumNextNotes-v0.90.5-beta-server-linux-arm64.tar.xz version?
